packages/ckeditor5-core/tests/editor/utils/dataapimixin.js

```diff
@@ -5,73 +5,13 @@
 
 import DataApiMixin from '../../../src/editor/utils/dataapimixin.js';
 import Editor from '../../../src/editor/editor.js';
-import testUtils from '../../../tests/_utils/utils.js';
-import { getData, setData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model.js';
 
 describe( 'DataApiMixin', () => {
-	let editor;
+	it( 'it returns the passed parameter for compatibility reasons', () => {
+		class CustomEditor extends Editor {}
 
-	beforeEach( () => {
-		// eslint-disable-next-line new-cap
-		class CustomEditor extends DataApiMixin( Editor ) {}
+		const editor = new CustomEditor();
 
-		editor = new CustomEditor();
-		editor.model.document.createRoot( '$root', 'main' );
-		editor.model.document.createRoot( '$root', 'secondRoot' );
-		editor.model.schema.extend( '$text', { allowIn: '$root' } );
-		editor.fire( 'ready' ); // (#6139)
-	} );
-
-	afterEach( async () => {
-		await editor.destroy();
-	} );
-
-	it( 'is compatible with `mix` function', () => {
-		expect( DataApiMixin ).have.property( 'setData' ).to.be.a( 'function' );
-		expect( DataApiMixin ).have.property( 'getData' ).to.be.a( 'function' );
-	} );
```

Comment:
I'd keep such a test to ensure we are backward compatible with the old `mix()` function.